modules/field_event_dispatcher/src/Event/Field/AbstractFieldSettingsSummaryFormEvent.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Drupal\field_event_dispatcher\Event\Field;
+
+use Drupal\hook_event_dispatcher\Event\EventInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * Class AbstractFieldSettingsSummaryFormEvent.
+ */
+abstract class AbstractFieldSettingsSummaryFormEvent extends Event implements EventInterface {
+
+  /**
+   * An array of summary messages.
+   *
+   * @var array
+   */
+  protected $summary;
```

Comment:
No need for this to be protected, it can safely be private as there is no real dependency for the child classes on this.